subcommand_plugin_examples/conda_subcommand_plugin_tutorial/ascii_graph/LICENSE

```diff
@@ -0,0 +1,27 @@
+BSD 3-Clause License
```

Comment:
Shouldn't we just move this license to the project root? Then all the examples are covered under the same license and there's no ambiguity.

Reply:
I've gone ahead and moved the license to the root level of the project.